src/commands/es/snapshots/dump.ts

```diff
@@ -0,0 +1,46 @@
+import { flags } from '@oclif/command'
+import { Client } from '@elastic/elasticsearch'
+
+import { Kommand } from '../../../common'
+
+export default class EsSnapshotsDump extends Kommand {
```

Comment:
I don't think we should use the term "dump" in this module since they are not dumps but snapshots which is quite different
```suggestion
export default class EsSnapshots extends Kommand {
```

Reply:
So the command should be called `create` to follow ES terminology: https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html

Reply:
I think this is a little confusing to just call it `EsSnapshots`

Since in this "snapshots" module we can create 2 types of thing

1. Repository (the holder of snapshots)
1. Snapshot (data)

I prefer option `EsSnapshotsCreate` since we have an `EsSnapshotsCreateRepository`

@alexandrebouthinon wdyt ?

Reply:
And also, shall we rename the Kommand -> es:snapshots:create ?

ping @Aschen @alexandrebouthinon

Reply:
Yes I agree with you for the naming, we might as well stick to the Elasticsearch documentation 👍